org.jonata.Subtitld.yml

```diff
@@ -151,10 +151,6 @@ modules:
               - "--enable-pic"
             build-options:
               arch:
-                aarch64:
-                  cxxflags: "-fPIC"
-                  config-opts:
-                    - "--disable-lto"
```

Comment:
that does nothing to fix the problem. if anything you make it harder to actually build for aarch64.

Reply:
I'm lost. What should I do? x86_64 is enough.

Reply:
why not building pyside from source?

Reply:
Can I cancel the build?